keyboards/diverge3/keymaps/iso_uk/config.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2017 IslandMan93
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`?

It's simpler, and less prone to user error, so if you could make this switch?

```suggestion
#pragma once
```

Reply:
I disagree that it is better to use #pragma once  as it is a non standard - compiler specific feature.
Also having the include guard is in keeping with the style in the default layout.

Reply:
`#pragma once` has been supported in all relevant compilers since at least 2005. It's the recommended way of doing include guards in QMK, so please use it.

Reply:
`#pragma once` has been supported in all relevant compilers since at least 2005. It's the recommended way of doing include guards in QMK, so please use it.